<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xxx客户端</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="encryBtn">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>354</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>加密</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>438</x>
      <y>352</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainText">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>10</y>
      <width>295</width>
      <height>373</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="plainText">
     <string>请输入待加密文本...</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="addressT">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>30</y>
      <width>167</width>
      <height>29</height>
     </rect>
    </property>
    <property name="plainText">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="portT">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>80</y>
      <width>75</width>
      <height>29</height>
     </rect>
    </property>
    <property name="plainText">
     <string>16689</string>
    </property>
   </widget>
   <widget class="QLabel" name="addressL">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>36</y>
      <width>37</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>地址</string>
    </property>
   </widget>
   <widget class="QLabel" name="portL">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>88</y>
      <width>37</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pFilepath">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>196</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="plainText">
     <string>/home/chen/maxwell/pFile1</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="cFilepath">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>260</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Abyssinica SIL&quot;;</string>
    </property>
    <property name="plainText">
     <string>/home/chen/maxwell/cFile1</string>
    </property>
   </widget>
   <widget class="QLabel" name="pFilepathL">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>172</y>
      <width>125</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>明文保存地址</string>
    </property>
   </widget>
   <widget class="QLabel" name="cFilepathT">
    <property name="geometry">
     <rect>
      <x>324</x>
      <y>238</y>
      <width>125</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>密文保存地址</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
